error.go

```diff
@@ -98,11 +98,11 @@ func (p PingResultError) Subsitute(old string, new string) string {
 	return strings.ReplaceAll(string(p), old, new)
 }
 
-func (p PingResultError) NoError() bool {
+func (p PingResultError) HasError() bool {
```

Comment:
oh. this one including the previous PR. maybe remove the previous one or clean up here :trollface:

Reply:
I'll rebase once the other one goes in